### PR TITLE
+ [doc] Add doc for timer module.

### PR DESCRIPTION
**Pull request by YorkShen**
+ [doc] Add doc for timer module.

### CODE REVIEW

**Comment by luics** [comment]
+1
@zshshr  pls merge it before releasing 0.7 tomorrow.

File: doc/modules/timer.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Timer
+
+
+## Summary
+
+Weex encapulates a series of APIs in order to start/stop a one time task or a repeated task at a fixed delay. Please be noted that this moudle **don't provide an accuracy dealy**. It provides best-effort delivery, but the actual dealy may still exceed the delay user wants if the corresponding thread is busy.
```
Comment (luics) [inline doc/modules/timer.md]: * " a one time " -> "a timer?"
* " Please be noted that this moudle **don't provide an accuracy dealy**" -> " Please note that this module **don't provide an accuracy delay**"

File: doc/modules/timer.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## API
+All timeout or interval in this module are measured in milliseconds.
+
+Also, timeout and interval should be a non-negative integer(the max of integer is 0x7FFFFFFF). If timeout or interval is negative, then it will be reset to zero, e.g. the task will be put in the task queue immediately.
+
+### setTimeout(fnId: number, timeout: number)
+Execute an one-time task at a fixed delay.
```
Comment (luics) [inline doc/modules/timer.md]: Execute a task once at a fixed delay.

File: doc/modules/timer.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+#### Arguments
+* `fnId` *(number)*: the task to be excuted.
```
Comment (luics) [inline doc/modules/timer.md]: fnId is a number?

**Comment by Jinjiang** [comment]
Nice job!

Actually this module is made for the polyfill of HTML5 timer APIs, not recommended to use directly. So I suggest we put a strong paragraph at the beginning to note this just like the `dom` module note in its doc. Additionally with some explanation of why we use `fnId: number` as a callback identifier is better.

Thanks.

File: doc/modules/timer.md (hunk 3 above)
Comment (YorkShen) [inline doc/modules/timer.md]: Yeah, it is a number according to https://github.com/alibaba/weex/issues/627

File: doc/modules/timer.md (hunk 1 above)
Comment (YorkShen) [inline doc/modules/timer.md]: I think "a one-time task" would be more appropriate.

File: doc/modules/timer.md (hunk 2 above)
Comment (YorkShen) [inline doc/modules/timer.md]: "Execute a task after a fixed delay." would be better.

**Pull request merged**